pkg/networkplugin-syncer/handlers/ovn/router_ovn_cluster_north.go

```diff
@@ -63,80 +60,6 @@ func (ovn *SyncHandler) connectOvnClusterRouterToSubm() error {
 	return nil
 }
 
-// changeMgmtAllowRelatedACL modifies an ACL installed by ovn-kubernetes, which in it's default
-//                           form will prevent ICMP fragment packets getting back to pods, precluding
-//                           Path MTU discovery from working
-func (ovn *SyncHandler) changeMgmtAllowRelatedACL(chassisSwitch string) error {
```

Comment:
This is removed because it was a workaround that stopped working ( see https://github.com/submariner-io/submariner/issues/1278 ) and we will try a different solution.